docs/build/quick-start/add-zksync-to-metamask.md

```diff
@@ -36,4 +36,14 @@ Fill in the following details for the zkSync network:
 
 Click on "Save" to add the zkSync network to MetaMask.
 
+## Import from Chainlist
+
+To add zkSync as a custom network in Metamask from Chainlist.org, follow these steps:
+
+- Visit https://chainlist.org/chain/324 (Mainnet) or https://chainlist.org/chain/300 (Testnet) on your browser
+- Click Connect
+- Unlock your metamask
```

Comment:
Can you clarify what you mean by "Unlock"? I think we can be more precise with a statement of `Connect your MetaMask wallet`, thoughts?